OrleansDashboard/ServiceCollectionExtensions.cs

```diff
@@ -26,6 +26,8 @@ public static class ServiceCollectionExtensions
         {
             builder.ConfigureApplicationParts(parts => parts.AddDashboardParts());
             builder.ConfigureServices(services => services.AddDashboard(configurator));
+            builder.AddIncomingGrainCallFilter<GrainInteractionFilter>();
```

Comment:
I think I would like a way to control (switch on/off) this profiling, as it may get slow?